chevah_build

```diff
@@ -998,7 +998,7 @@ command_test_compat() {
     # the local version.
     echo '##### Running chevah.compat tests... #####'
     execute rm -rf compat
-    execute git clone https://github.com/chevah/compat.git --depth=1 -b master
+    execute git clone https://github.com/chevah/compat.git --depth=1 -b 5509-cryptography32-openssl102
```

Comment:
the code is in master now :)

Reply:
Thanks, I was waiting to see all OS'es run `compat` tests fine. Happy to have found the little fix required to fix them on AIX.